<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Principal</class>
 <widget class="QMainWindow" name="Principal">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1256</width>
    <height>650</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindows</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnNuevo">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Nuevo</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>40</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Actividades: </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="tablaView">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>130</y>
      <width>781</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
      <underline>false</underline>
      <strikeout>false</strikeout>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>D i,j</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>O i,j</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ti0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ti1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tj0</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tj1</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>MT i,j</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Ml i,j</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha inicio tempano</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignTop</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fecha inicio tardio</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>fecha final temprano</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
    <column>
     <property name="text">
      <string>fecha final tardio</string>
     </property>
     <property name="textAlignment">
      <set>AlignJustify|AlignVCenter</set>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="tabla">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>130</y>
      <width>361</width>
      <height>391</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <column>
     <property name="text">
      <string>Actividad</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Detalle</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Predecesora</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>To</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tn</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Tp</string>
     </property>
    </column>
   </widget>
   <widget class="QSpinBox" name="canVariable">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>40</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
   </widget>
   <widget class="QPushButton" name="btnCalcular">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>540</y>
      <width>91</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Calcular</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnGenera">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>70</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Century Gothic</family>
      <pointsize>9</pointsize>
      <weight>50</weight>
      <bold>false</bold>
     </font>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Generar</string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionSimplex">
   <property name="text">
    <string>Simplex</string>
   </property>
  </action>
  <action name="actionPerl">
   <property name="text">
    <string>Pert</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
